microsoft-edge/webview2/samples/webview2apissample.md

```diff
@@ -72,12 +72,30 @@ To use Visual Studio 2017, you might also need to install a recent Windows SDK o
 <!-- ====================================================================== -->
 ## Step 4 - Open the solution in Visual Studio
 
-1. On your local drive, open the `.sln` file in Visual Studio:
+1. Open Visual Studio 2022.  (Or, if you have Visual Studio 2019 installed, you can use that version.)  For example, in Windows, in the Start menu, search for **Visual Studio 2022** app, and then click **Open**:
 
-   *  `<your-repos-directory>/WebView2Samples/SampleApps/WebView2Samples.sln`
+   ![Start menu showing variants of "Visual Studio 2022", including the app to open](./webview2apissample-images/open-vs2022.png)
 
-   or:
+1. In the **Open recent** dialog, click **Continue without code**, in the lower right:
+
+   ![Continue without code, when starting Visual Studio 2022](./webview2apissample-images/continue-without-code.png)
+
+1. Click the **Git** menu in the upper left, point to **Local Repositories**, and then select **WebView2Samples**.
+
+1. In the lower right, next to the **Solution Explorer** tab, click the **Git Changes** tab, click the down arrow next to the **main** branch name, and then click the **New Branch** button:
+
+   ![Creating a new working branch of the repo, from within Visual Studio](./webview2apissample-images/git-new-branch.png) 
+
+1. In the **Create a new branch** dialog, in the **Branch name** field, enter a branch name, such as `user/<myGitHubUsername>/WebView2Samples`, and then click the **Create** button:
 
+   ![The "Create a new branch" dialog](./webview2apissample-images/create-a-new-branch.png)
+
+1. Click the **Solution Explorer** tab, in the lower right, and then in **Solution Explorer**, double-click **WebView2Samples.sln**.
+
+   Or, the following approach requires more navigation: Select **File** > **Open** > **Project/Solution** > `WebView2Samples.sln`, such as (on Windows): `C:\Users\myUsername\GitHub\WebView2Samples\SampleApps\WebView2Samples.sln`
+
+   Example paths where you may have cloned the WebView2Samples repo to:
+   *  `<your-repos-directory>/WebView2Samples/SampleApps/WebView2Samples.sln`
    *  `<your-repos-directory>/WebView2Samples-main/SampleApps/WebView2Samples.sln`
```

Comment:
Actually, I think the following steps are not quite correct. I agree that the original description was a little bit confusing and could be clarified. 

I think the general steps can be summarized be as follows:
1. Open Visual Studio 2022
2. Select `Open a project or solution`
3. Navigate to where you cloned the WebView2Samples repository, and select the `WebView2Samples.sln`. E.g. `<some-directory>\WebView2Samples\SampleApps\WebView2\WebView2Samples.sln`

Reply:
reverted changes in PR other than debug article, will re-apply/ followup in other PR